common-api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -143,6 +145,7 @@ public static WorkbenchConfig createEmptyConfig() {
     // Controls whether all api requests are traced and sent to Stackdriver tracing, or
     // whether we only trace at the default frequency.
     public boolean traceAllRequests;
+    public String location;
```

Comment:
Hmm i will be looking at jay's PR and if there does seem to be a need to maintain 2 locations i would prefer to name it something more specific to cloud task queue since it _might_ just be used for that